app/src/main/res/layout/fragment_channel.xml

```diff
@@ -6,165 +6,173 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <ScrollView
-        android:id="@+id/channel_scrollView"
+    <androidx.coordinatorlayout.widget.CoordinatorLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:orientation="vertical"
-        android:visibility="invisible"
-        tools:context=".ui.fragments.ChannelFragment">
+        android:layout_height="match_parent">
 
-        <LinearLayout
+        <com.google.android.material.appbar.AppBarLayout
+            android:id="@+id/channel_app_bar"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:orientation="vertical">
+            android:layout_height="wrap_content">
 
-            <ImageView
-                android:id="@+id/channel_banner"
+            <com.google.android.material.appbar.CollapsingToolbarLayout
```

Comment:
Why do we introduce a collapsing toolbar here although we're not actually collapsing anything?

Reply:
-the content above the list of videos needs to be scrollable. The CT layout provides that behavior to its children in synchrony with other components like Recycler view. If content is placed inside app bar layout without being wrapped in the CT, it will remain fixed and not scroll up. Although in CT, it does not appear as if the content is collapsing it is possible that the container is and the content is remaining fixed as it is cut off in view as you scroll.
-an alternative to using CT would be to implement adapters that inflate different types of views so that the top views above the list can scroll off the screen in the same container as the list videos. 
-I feel that implementing CT in xml is easier and avoids adding more programmatic complexity to recyclerview adapters.